python/langwatch/src/langwatch/guardrails.py

```
@@ -0,0 +1,46 @@
+from typing import List, Optional
+
+import langwatch.evaluations
+from langwatch.types import Conversation, RAGChunk
+
+
+def evaluate(
+    slug: str,
+    input: Optional[str] = None,
+    output: Optional[str] = None,
+    expected_output: Optional[str] = None,
+    contexts: List[RAGChunk] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)